deis-dev/manifests/deis-minio-rc.yaml

```diff
@@ -18,8 +18,24 @@ spec:
         - name: deis-minio
           image: quay.io/deisci/minio:v2-beta
           imagePullPolicy: Always
+          env:
+            - name: HEALTH_SERVER_PORT
+              value: "8082"
           ports:
             - containerPort: 9000
+            - containerPort: 8082
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8092
```

Comment:
fixed in https://github.com/arschles/deis-charts/commit/eed9e8d8f7e851786746aee998673332674f747d